usr/share/rear/output/RAWDISK/Linux-i386/280_create_bootable_disk_image.sh

```diff
@@ -187,6 +187,37 @@ losetup -d "$disk_device" || Error "Could not delete loop device"
 RemoveExitTask "losetup -d $disk_device >&2"
 
 
+### Copy the EFI boot partition to local disk partitions named "$RAWDISK_INSTALL_GPT_PARTITION_NAME" if configured
+
+if [[ -n "$RAWDISK_BOOT_EFI_STAGING_ROOT" && -n "$RAWDISK_INSTALL_GPT_PARTITION_NAME" ]]; then
+    local detected_fs_type detected_mount_point install_partition install_partition_count=0
+
+    LogPrint "Installing the EFI rescue system to partitions labeled '$RAWDISK_INSTALL_GPT_PARTITION_NAME'"
+
+    for install_partition in $(lsblk --paths --list --noheadings --output=NAME,PARTLABEL | awk "/ $RAWDISK_INSTALL_GPT_PARTITION_NAME"'$/ { print $1; }'); do
```

Comment:
Older 'lsblk' may not support the options you need here.
I suggest to not care about adding support also of such old systems
and to error out when 'lsblk' failed so the user is informed that
this functionality is not supported on his old system.

Reply:
Added checks for critical `lsblk` invocations inside the loop via 1efc314.

If the loop's initial "partition finder" fails to find suitable partitions because of an ancient `lsblk` version it will be treated just like any other reason (e.g. no suitably named partitions present) and the user will see a message "Could not detect suitable partitions labeled ...". This would be detected during ReaR testing anyway.